### ISSUE TITLE
Players roaming when walking out of bounds

### ISSUE THREAD

**Issue by nessa-art** [olejarza/VACATe#106]
**Description**
When players walk off the platforms/playgrounds,  they drift downwards or they're able to walk back onto the platforms. Issue is  not specific to any level/scene.

**Steps to Reproduce**
Start level 1 and walk off the level platforms.

**Expected Behaviour**
Players should fall downwards with acceleration and die.

**Actual Behaviour**
Player is slowly drifting and is able to walk around in empty space.

**Comment by camvr** [comment]
Yeah I had to remove player "gravity" since it was super bugged (hence that weird slow fall, which was actually just strange floating point errors causing the player position to very slowly move down the y-axis), I'll try to get around to this for implementation 1
